Examples/onnx/quantization/quantsim.ipynb

```diff
@@ -2,50 +2,64 @@
  "cells": [
   {
    "cell_type": "markdown",
-   "metadata": {
-    "collapsed": false,
-    "jupyter": {
-     "outputs_hidden": false
-    }
-   },
+   "metadata": {},
    "source": [
-    "# Quantization Simulation\n",
+    "# Quantization-aware training\n",
```

Comment:
We shouldn't call it `Quantization-aware training`. Keep `Quantization simulation`. We don't have QAT techniques in aimet-onnx